assets/home/tasks-section/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Internal dependencies
+ */
+import Section from '../section';
+import Progress from './progress';
+import Tasks from './tasks';
+import FirstCourse from './first-course';
+import Ready from './ready';
+
+/**
+ * Tasks section component.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Tasks data.
+ */
+const TasksSection = ( { data } ) => {
+	const items = Object.values( data.items );
+
+	const sortedItems = items.sort( ( a, b ) => a.priority - b.priority );
+	const completedItems = sortedItems.filter( ( i ) => i.done );
+
+	return (
+		<Section
+			className="sensei-home-tasks-section"
+			insideClassName="sensei-home-tasks-section__inside"
+		>
+			{ data.is_completed ? (
+				<Ready />
+			) : (
+				<>
+					<div className="sensei-home-tasks-section__content">
+						<Progress
+							totalTasks={ items.length }
+							completedTasks={ completedItems.length }
+						/>
+						<Tasks items={ sortedItems } />
+					</div>
+					<div className="sensei-home-tasks-section__first-course">
+						<FirstCourse
+							siteTitle="Learn Photography"
+							courseTitle="Architectural Photography"
+							siteLogo="https://techcrunch.com/wp-content/uploads/2022/06/Leica-on-black.jpeg?w=1390&crop=1"
+							featuredImage="https://techcrunch.com/wp-content/uploads/2022/06/Leica-on-black.jpeg?w=1390&crop=1"
```

Comment:
Just a question that we had while implementing #5936: do these images need to be of a given specific size? 

I see here that the images are both ~1390px of width, but, I'm not sure if there's a minimum/maximum value that I should request from WordPress.

Reply:
I checked that the biggest largest width we use is `834px` (when it breaks to the bottom of the tasks list). I'm not sure about the default sizes that WP offers, but I think the next one bigger that this would be good.

Reply:
Well, I merged #5936 already, but it might be a good idea to review later to check if the sizes are appropriate. Not sure if I should create an issue to do that, though.